charts/calling-test/templates/NOTES.txt

```diff
@@ -0,0 +1,5 @@
+1. Get the application URL by running these commands:
+
+  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "calling-test.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
+  echo "Visit http://127.0.0.1:8080 to use your application"
+  kubectl --namespace {{ .Release.Namespace }} port-forward $POD_NAME 8080:8080
```

Comment:
Is this needed given that the application is available over `https://BACKEND_URL/calling-test`?

Reply:
It's only available that way when wire-server is installed. If installing this chart on your own (and e.g. pointing to a completely different server), it's still a useful way of testing that it works without needing full wire-server and nginz.